Controller/admin_login_signup.js

```diff
@@ -0,0 +1,53 @@
+const Admin = require("../Model/AdminSchema");
+const bcrypt = require("bcrypt");
+const jwt = require('jsonwebtoken');
+const jwtSecret = "qwerty131";
+
+
+const signUp = async(req, res) => {
+  try {
+    console.log(req.body);
+    const admin = new Admin({
+        Admin_Account_Address : req.body.Admin_Account_Address,
+        First_Name : req.body.First_Name,
+        Last_Name : req.body.Last_Name,
+        EmailId : req.body.EmailId,
+        Password :  await bcrypt.hash(req.body.Password, 10),
+      });
+      await admin.save();
+      res.send("Admin signup Successful");
+  } catch (error) {
+    console.log(error);
+    res.status(400).send(error);
+  }
+  
+}
+
+
+
+const login = async (req, res) => {
+    try {
+      const { EmailId, Password } = req.body;
+      if (!EmailId || !Password) {
+        return res.status(400).send({ error: "Plz enter the data" });
```

Comment:
`Plz ` should be `Please`

Reply:
Did the change with Please instead on Plz